### PR TITLE
Default OCP and OCS versions to 4.5

### PR DESCRIPTION
**Pull request by clacroix12**
Fixes: #2253 

Signed-off-by: Coady LaCroix <clacroix@redhat.com>

### CODE REVIEW

File: ocs_ci/framework/conf/default_config.yaml [hunk 1]
```diff
@@ -120,7 +120,7 @@ ENV_DATA:
 # This section is related to upgrade
 UPGRADE:
   upgrade_to_latest: true
-  ocp_channel: "stable-4.4"
+  ocp_channel: "stable-4.5"
```
Comment (petr-balogh) [inline ocs_ci/framework/conf/default_config.yaml:123]: I don't think such channel exists when OCP 4.5 is not GAed yet.

@AviadP can tell you more details what to use but I guess we should use something similar what we had here:
https://github.com/red-hat-storage/ocs-ci/blob/master/conf/ocp_version/ocp-4.4-candidate-upgrade.yaml 

Most likely there will be already candidate build for 4.5 but haven't checked here.

Other option to use nightly like in this config:
https://github.com/red-hat-storage/ocs-ci/blob/master/conf/ocp_version/ocp-4.4-upgrade.yaml

Which will be probably better option, but for this you will need to change also one line below.

Comment (petr-balogh) [reply ocs_ci/framework/conf/default_config.yaml:123]: Please check the code how it is used or wait for @AviadP explanation. I do not remember it either. Just brought this up for discussion as we need to set it correctly for OCP upgrade.

Comment (AviadP) [reply ocs_ci/framework/conf/default_config.yaml:123]: @petr-balogh, channel "cnadidate-4.5" should be ready for use. as far as i understand, "stable-4.5" isnt
i will create config files for 4.5-nightly or 4.5 candidate

Comment (clacroix12) [reply ocs_ci/framework/conf/default_config.yaml]: `ocp_channel` is now set to `candidate-4.5`.

File: ocs_ci/framework/conf/default_config.yaml [hunk 2]
```diff
@@ -101,12 +101,12 @@ ENV_DATA:
...
   skip_ocp_deployment: false
   skip_ocs_deployment: false
-  ocs_version: '4.4'
+  ocs_version: '4.5'
   # uncomment to use custom directory for storing measurement data related to
   # monitoring tests, otherwise generate temporary directory for each test run
   # measurement_dir: '/tmp/ocs_ci_monitoring_measurement/'
   # Default RHCOS image to be used for VmWare deployment
-  vm_template: 'rhcos-4.4.0-rc.1-x86_64-vmware.x86_64'
+  vm_template: 'rhcos-4.5.0-rc.1-x86_64-vmware.x86_64'
```
Comment (vavuthu) [inline ocs_ci/framework/conf/default_config.yaml:109]:
```suggestion
  vm_template: ' rhcos-4.5.0-0.nightly-2020-03-18-115438-x86_64-vmware.x86_64'
```

Comment (clacroix12) [reply ocs_ci/framework/conf/default_config.yaml:109]: @vavuthu is the space at the start of the template string necessary or a typo?

Comment (vasukulkarni) [reply ocs_ci/framework/conf/default_config.yaml]: i think just a typo

Comment (clacroix12) [reply ocs_ci/framework/conf/default_config.yaml]: I updated the vm_template without the space at the start, assuming it was a typo.

Comment (vavuthu) [reply ocs_ci/framework/conf/default_config.yaml]: its typo

**Pull request merged**